parameters.d/osg34-stashcache.yaml

```diff
@@ -0,0 +1,47 @@
+###################
+# OSG 3.4 StashCache tests
+# File format documention:
+# https://github.com/opensciencegrid/vm-test-runs#running-osg-test-in-vm-universe
+###################
+
+platforms:
+  - centos_7_x86_64
+  - sl_7_x86_64
+
+sources:
+  ###################
+  # Format:
+  # [<Github account>:<osg-test branch>;] <OSG ver>; <REPO 1, REPO 2...REPO N> [> <Update OSG ver>/<Update REPO 1, REPO 2...REPO N>]
+  # Example:
+  # Run osg-test with packages from 3.2-release and 3.2-testing that are then upgraded to 3.3-testing and 3-3-upcoming-testing:
+  # 3.2; osg, osg-testing > 3.3/osg-testing, osg-upcoming-testing
+  ###################
+  # - opensciencegrid:master; 3.4; osg
+  - opensciencegrid:master; 3.4; osg-testing
+  - opensciencegrid:master; 3.4; osg-testing > osg-development
+  - opensciencegrid:master; 3.4; osg-development
+
+package_sets:
+  #### Required ####
+  # label - used for reporting, should be consistent across param files
+  # packages - list of packages to install in the test run
+  #### Optional ####
+  # selinux - enable SELinux for the package set, otherwise Permissive mode (default: True)
+  # osg_java - Pre-install OSG java packages (default: True)
+  # rng - Install entropy generation package (default: False)
+  ##################
+  - label: All + StashCache
+    osg_java: False
+    packages:
+      - osg-tested-internal
+      - stashcache-client
+      - stashcache-origin-server
+      - stashcache-cache-server
+      - stashcache-cache-server-auth
```

Comment:
Why do we need to test the case with `osg-tested-internal`? Are we planning on supporting other services installed alongside stashcache?

Reply:
Well, I didn't notice that the tests + xrootd needed fixing until I installed both side-by-side.

Reply:
They collided by both running on the same port.

Reply:
That goes back to my point: do we expect to support a standalone XRootD installation alongside an XCache? I'm not sure why anyone would run that way

Reply:
Well, we don't expect anyone to install everything in osg-tested-internal either yet we still install it all in the nightlies. I'm hoping to eventually put SC into osg-tested-internal anyway.

(Did you know that at one point it was possible to have an osg-test run with zero skips? We even have code to mark a "perfect" run. Guess what's never going to happen again...)

Reply:
It's not a biggie though. If you don't want it in there, I can take it out.